src/server/apollo/dataSources.ts

```diff
@@ -21,6 +21,7 @@ const LAST_UPDATE_CODE = '{{lastUpdate}}';
 /**
  * CustomAPI class to create a dataSource fetching from an APIConfiguration.
  * If nothing is passed in the constructor, it will only be a standard REST DataSource.
+ * Datasources are called for example when using Reference Datas in a context where displaying data, for example in a grid or fetching/downloading records history items
```

Comment:
Please check if this is accurate or improve description

Reply:
@Matthis-M-ReliefApps 
that's a lot of "for example" but I think that's fine ;)